doc/configure/access_log_directives.html

```diff
@@ -174,15 +174,19 @@ <h3><a href="configure/access_log_directives.html#access-log"><code>"access-log"
 <tr id="total-time"><td><code>total-time</code><td>same as <code>duration</code> (since v2.3)
 </table>
 <table>
-<caption>Proxy Timings (since v2.3)</caption>
+<caption>Proxy (since v2.3)</caption>
```

Comment:
Actually, I think you might want to make "Proxy Timings" and "Proxy" different tables. Or move proxy-*-time into the "Access Timings" table.

What I am suggesting here is that proxy-*-time should be listed closely to the variables defined by "Access Timing", as their role is similar (and naming convention too).